web/components/Canvas/Canvas.module.scss

```diff
@@ -1,20 +1,9 @@
 .Canvas {
   display: flex;
   justify-content: center;
-  background-color: black;
-  width: 100%;
-  height: 100%;
+  background-color: whitesmoke;
+  min-width: 100vh;
+  min-height: 100vh;
   padding: 0;
   text-align: center;
-
-  .box {

```

Comment:
pull this back in for your next pr with position absolute if you want your canvas to just be the whole background which is what this looks like